app/controllers/users_controller.rb

```diff
@@ -10,7 +10,7 @@ def index
       
       def show
         @user = User.find(params[:id])
-        @courses = Course.all
```

Comment:
Shouldn't this stay Course.all? How would a student be added to a course if they can't see it to join it? 
Is a different joining mechanic being used instead?

Reply:
Students can add the course on the /courses page. The My Courses page (/user) should only show your own courses

Reply:
Ah gotcha, thanks for clarifying! Will resolve and merge : )